src/rust/kafka-metrics-exporter/tests/integration_tests.rs

```diff
@@ -21,13 +22,17 @@ mod integration_tests {
     };
     use tokio_stream::StreamExt;
 
+    fn brokers() -> String {
```

Comment:
Shouldn't this be defined in grapl-config/src/lib.rs and then called here? Also can we rename this to be kafka_brokers?

Reply:
Good call, yes.